src/main/java/com/redhat/devtools/intellij/tektoncd/tkn/TknCli.java

```diff
@@ -170,4 +176,35 @@ public void editCustomResource(KubernetesClient client, String namespace, String
     public void createCustomResource(KubernetesClient client, String namespace, CustomResourceDefinitionContext crdContext, String objectAsString) throws IOException {
         client.customResource(crdContext).create(namespace, objectAsString);
     }
+
+    public void startTask(String namespace, String task, Map<String, String> parameters, Map<String, String> inputResources, Map<String, String> outputResources) throws IOException {
+        List<String> args = new ArrayList<>(Arrays.asList("task", "start", task, "-n", namespace));
+        args.addAll(argsToList(parameters, inputResources, outputResources));
+        ExecHelper.execute(command, args.toArray(new String[0]));
+    }
+
+    private List<String> argsToList(Map<String, String> parameters, Map<String, String> inputResources, Map<String, String> outputResources) {
+        List<String> args = new ArrayList<>();
```

Comment:
There is probably some refactoring to be done here as each block is the same with the exception of the prefix so those blocks can be refactored as a separate method with prefix parameter and the map